src/resolvers/FlyerResolver.ts

```diff
@@ -238,6 +243,32 @@ class FlyerResolver {
       startDate,
       title,
     );
+    let editedResponse = '';
+    if ((endDate != null && location != null) || (location != null && startDate != null)) {
+      editedResponse = editedResponse.concat('changed its date and location');
+    } else if (endDate != null && startDate != null) {
+      editedResponse = editedResponse.concat('changed its date');
+    } else if (endDate != null) {
+      const date = new Date(endDate);
+      editedResponse = editedResponse.concat(
+        'changed its end date to '.concat(date.toDateString()),
```

Comment:
Can we add comments describing each `if` branch as well as what `.toDateString()` formats it to?